app/services/end_of_cycle_timetable.rb

```diff
@@ -54,8 +54,12 @@ def self.between_cycles_apply_2?
   end
 
   def self.date(name)
-    if HostingEnvironment.test_environment? && FeatureFlag.active?(:simulate_time_between_cycles)
-      return simulate_time_between_cycles_dates[name]
+    if HostingEnvironment.test_environment? || HostingEnvironment.sandbox_mode?
+      if FeatureFlag.active?(:simulate_time_between_cycles)
+        return simulate_time_between_cycles_dates[name]
+      elsif FeatureFlag.active?(:simulate_time_mid_cycle)
```

Comment:
These flags should really be mutually exclusive — I wonder if there's a way to enforce that. This solves the immediate problem tho

Reply:
I did wonder about that but I don't think these switches should be feature flags in the longer term. We could add validation to the `FeatureFlag` model to enforce it? Going to merge for now but happy to add that as a follow-up if you think worthwhile.